### DISCUSSION TITLE
Forcing `render` on specific `Item`s or disabling filter for specific groups

### DISCUSSION THREAD

**Discussion by mrmartineau** [pacocoursey/cmdk#62] [Ideas]
I am using `cmdk` as a way to search and navigate my app. In most cases I'd like to use the built-in filtering that provided with the package, but for some `cmdk` groups (as I have them), I'd like to not use the built-in filtering and instead show them based on an API response from my app's search endpoint - it's kind of like the `shouldFilter=false` behaviour except only for specific items or groups.

Having looked through the code, I think that the `render` functions (in `Item` and `Group`) decides whether to show the item or not. I think adding a `forceRender` boolean prop would solve the issue on both those elements. Would that be the right way to go about it?

```diff
const render = useCmdk((state) =>
+     forceRender ? true
      : context.filter() === false ? true
      : !state.search ? true
      : state.filtered.groups.has(id)
    )
```

I'm happy to submit a PR for the changes, but wanted to see if there was already a way to do it

**Comment by SorenHolstHansen** [comment]
I would like this too!

**Comment by pacocoursey** [comment]
Let's do it! If you're open to submitting a PR @mrmartineau I'd be happy to review.

Let's call the prop `forceMount`, to match Radix UI prop standards.

**Reply by MildTomato** [reply]
+1 for forceMount

**Reply by MildTomato** [reply]
@pacocoursey here you go 😄 
https://github.com/pacocoursey/cmdk/pull/101

**Comment by mrmartineau** [comment]
Fixed in #101
